activities/trainings/writing-issues.md

```diff
@@ -0,0 +1,73 @@
+---
+type: Guide
+explains: How to clearly communicate what is wrong and suggest fixes
+---
+
+# Writing issues
+
+An issue is a story explaining why something should be changed. It serves as the seed for collaboration in order to get it solved.
+
+Issues can be of two types:
+
+* Bug: something that isn't as it is supposed to be, this is broken
+* Enhancement: an improvement that could be done to make it better
+
+The goal of an issue is to provide knowledge about and context on what to fix or improve. You can make issues alone or together.
+
+An issue can contain a proposed fix, but does not have to.
+
+We write good issues because:
+
+* it helps communicate things that need to be done
+* planning becomes easier if you understand the context of a change
+* the person filing an issue is not always the person to fix it, they might have different skillsets
+* commit messages become better with more context
+
+We use the list of issues to:
+
+* understand what needs to be done
+* prioritize what to do
+* collaborating with each other on finding solutions
+* help new contributors find out how they can contribute
+
+## Structure
+
+The title of the issue should be short, to the point an descriptive. Enhancements can have verbs in them. Detail, technical or otherwise, should be in the body of the issue.
+
+* Context: why are you filing this issue
+* Problem or idea
+* Solution or next step: request feedback from others, propose changes or propose further investigation
+
```

Comment:
I've reordered the sections, changed the hierarchy and added more info on adding standard labels.

```suggestion
## Defining done for issues

It can help those who solve the issue to know when they've actually fixed it. If there are multiple factors in this it might help adding them as checkboxes at the end of the issue, then they can be resolved and tracked one by one.

## Tag relevant contributors and other issues

Context is key, so use the GitHub functionality of tagging other contributors when relevant as well as mentioning other issues and pull requests. Add any relevant labels.

## Specific types of issues

We use these standard labels to classify issues:

 * bugs
 * enhancements
 * good first issue 
 
 Each type of issue has information is normally included, effectively acting as a template. 
 Always add a label if you make one of these issues.

### How to write a bug

A bug is something that is wrong. In order to clearly communicate what is wrong it helps to answer the following questions:

* What did you try to do or find out
* What did you expect
* What did you find

### How to write enhancements

An enhancement is something that could be better. In order to understand what the enhancement really is it can help to specify:

* a motivation: what users will benefit from this
* a description of how you envision this to be or how you would solve this

### How to write good first issues

Issues labeled `good first issue` provide an easy way for new contributors to get into committing. 

A good first issue describes the issue as well as providing a step by step instruction on how to resolve it.

## Further reading

* ["Writing meaningful GitHub issues results in better commit messages" by Michael Thorne](https://www.userx.co.za/journal/writing-good-github-issues-makes-for-better-commit-messages/)
* =["How we write our GitHub issues" by Wirecraft](https://wiredcraft.com/blog/how-we-write-our-github-issues/)
```

Reply:
I don't think this works with a 'suggestion', which is one line, committing this will replace the one line with all of these things, resulting in duplicate content.

As a note of order, you've added some spaces in front of asterisks and one `=` before the link at the end. I'll clean up and resolve.

Reply:
Ow, and also reading the content I see that I need to explain more. I've mentioned bugs and enhancements as the two types of issues, not necessarily meant as labels. All issues are one of these two types, regardless of what other taxonomy we add to it. It is either something which is broken, or something which can be improved.

Reply:
I'll go figure out how to integrate the two.